src/DatasourceWithAsyncBackend.ts

```diff
@@ -118,48 +119,43 @@ export class DatasourceWithAsyncBackend<
           const query: TQuery & DataQueryMeta = {
             ..._query,
             meta: { queryFlow: 'async' },
+            intervalMs,
+            maxDataPoints,
+            // getRef optionally chained to support < v8.3.x of Grafana
+            datasource: this?.getRef(),
+            datasourceId: this.id,
+            ...this.applyTemplateVariables(_query, request.scopedVars),
           };
 
-          const data = {
-            queries: [
-              {
-                ...query,
-                intervalMs,
-                maxDataPoints,
-                // getRef optionally chained to support < v8.3.x of Grafana
-                datasource: this?.getRef(),
-                datasourceId: this.id,
-                ...this.applyTemplateVariables(query, request.scopedVars),
-              },
-            ],
-            range: range,
-            from: range.from.valueOf().toString(),
-            to: range.to.valueOf().toString(),
-          };
-
-          let headers = {};
+          // Manually bypass the query cache for running queries if the caching service is not enabled.
+          // The caching service handles bypassing the query cache automatically when it is enabled.
           const cachingDisabled = !config.featureToggles.awsAsyncQueryCaching;
           if (cachingDisabled && isRunning(status)) {
-            // bypass query caching for Grafana Enterprise to
-            // prevent an infinite loop
-            headers = { 'X-Cache-Skip': true };
+            const requestSupportsSkipQueryCache = gte(config.buildInfo.version, '10.2.3');
```

Comment:
nit: It may be a little clearer to read to get `lte` and then check for the boolean (without negating it)